### ISSUE TITLE
[wg/webappsec] Web Application Security WG 2023

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#426]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:](https://htmlpreview.github.io/?https://github.com/w3c/webappsec/blob/main/admin/webappsec-charter-2023.html
)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 
 - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F06%2Fwebappsec-charter-2022.html&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fwebappsec%2Fmain%2Fadmin%2Fwebappsec-charter-2023.html), and any issue discussion: (none yet, see   [webappsec repo](https://github.com/w3c/webappsec/issues))

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 
 None

Known or potential areas of concern: 
 None

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
  [webappsec repo](https://github.com/w3c/webappsec/issues)

Anything else we should think about as we review? 
 Nope

cc @mikewest @dveditz

**Comment by plehegar** [comment]
Advance notice: https://lists.w3.org/Archives/Member/w3c-ac-members/2023JulSep/0026.html

**Comment by mikewest** [comment]
We discussed the rechartering at TPAC, noting a few additions and removals from our deliverables: https://github.com/w3c/webappsec/blob/main/meetings/2023/2023-09-15-TPAC-minutes.md#rechartering.

**Comment by plehegar** [comment]
Pull requests for the comments from TPAC 2023: https://github.com/w3c/webappsec/pull/635

**Comment by himorin** [comment]
- the latest entries of history table does not match with current running one at https://www.w3.org/2022/06/webappsec-charter-2022.html
- expected completion statement at 3. Deliverables seems to be needs-updated...? (should match with text in success criteria)
- (typo-ish) dt for `Securer Contexts` shall have class spec

**Comment by himorin** [comment]
no comment or request from i18n

**Comment by plehegar** [comment]
Security and privacy model for cookies ,  Permissions best practices and APIs, and  End-to-End Encryption email should be coordinated with the Privacy IG/WG.

**Comment by plehegar** [comment]
(from PING) some timelines are in 2022....

**Comment by ruoxiran** [comment]
no comments from APA.

**Comment by siusin** [comment]
The charter history is not yet completed. At least new deliverables like _Passkey Endpoints Well-Known URL_ should be mentioned as changes of this version.

**Comment by svgeesus** [comment]
> But then, in Deliverables, _both_ options are removed!
> 
> > > Choose one:
> > > Expected completion indicates when the deliverable is projected to become a Recommendation, or otherwise reach a stable state
> > > The Working Group intends to publish the latest state of their work as Candidate Recommendation (with Snapshots) and does not intend to advance their documents to Recommendation .

I notice, in the changes for the _previous_ charter:

> Moved most specs to snapshot (evergreen) publication.

so please add back

> The Working Group intends to publish the latest state of their work as Candidate Recommendation (with Snapshots) and does not intend to advance their documents to Recommendation .

**Comment by plehegar** [comment]
I fixed the charter. see https://github.com/w3c/webappsec/pull/641/files

**Comment by plehegar** [comment]
and https://github.com/w3c/webappsec/pull/642/files

**Comment by plehegar** [comment]
Charter review started:
  https://lists.w3.org/Archives/Public/public-new-work/2024Feb/0000.html

Deadline is 2024-03-02.

**Comment by plehegar** [comment]
We received 2 requests for changes, including one substantive.

**Comment by plehegar** [comment]
@marcoscaceres , is there an actual proposal that we can link from the WebAppSec charter ?

**Comment by plehegar** [comment]
status: there is an unforeseen delay on this, the propose changes won't come out until April 3rd.

**Comment by plehegar** [comment]
Following the AC Review, we are proposing the following changes

1. Remove "Off-The-Record Response Header Field" from the charter. It will be proposed as an addition to the Privacy Working Group separately.

2. Remove "End-to-end encryption email" from the charter. This was lacking an actual proposal and might be added in a future revision of the charter.

* [Proposed charter](https://www.w3.org/2024/04/proposed-wg-webappsec.html)
* [diff with previous proposed charter](https://services.w3.org/htmldiff?doc1=https://www.w3.org/2024/01/proposed-wg-webappsec.html&doc2=https://www.w3.org/2024/04/proposed-wg-webappsec.html)


Deadline to comment on those proposed changes is April 17.

https://lists.w3.org/Archives/Member/member-charters-review/2024Apr/0000.html

**Comment by plehegar** [comment]
no additional comments were received. Next step is for W3C to announce the new charter.

**Comment by simoneonofri** [comment]
Although the charter has already been revised, security is embedded in the fact that this is a group that develops security standards. Therefore, security is part of its mission statement.

Of course, from a methodological point of view, even a security feature can lead to additional security problems (such as the bad lock example in OSSTMM).
